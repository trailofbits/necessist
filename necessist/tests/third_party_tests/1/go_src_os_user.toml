url = "https://github.com/golang/go"
rev = "go1.23.0"
framework = "go"
full = true

subdir = "src/os/user"
target_os = "linux"
