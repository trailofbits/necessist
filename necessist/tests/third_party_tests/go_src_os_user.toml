url = "https://github.com/golang/go"
rev = "go1.20.1"
framework = "golang"
full = true

subdir = "src/os/user"
target_os = "linux"
