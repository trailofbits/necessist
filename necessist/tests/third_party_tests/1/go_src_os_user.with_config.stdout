16 candidates in 12 tests in 5 source files
$DIR/src/os/user/lookup_unix_test.go: dry running
$DIR/src/os/user/lookup_unix_test.go: mutilating
$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestFindGroupName`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""


This may indicate a bug in Necessist. Consider opening an issue at: https://github.com/trailofbits/necessist/issues

Silence this warning with: --allow run-test-failed
$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestFindGroupId`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""

$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestInvalidUserId`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""

$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestLookupUserId`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""

$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestLookupUserPopulatesAllFields`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""

$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestLookupUserPopulatesAllFields`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t(cached) [no tests to run]
```

stderr=""

$DIR/src/os/user/lookup_unix_test.go: Warning: Failed to run test `TestLookupUser`: code=0
stdout=```
testing: warning: no tests to run
PASS
ok  \tstd/os/user\t[..]s [no tests to run]
```

stderr=""

$DIR/src/os/user/user_test.go: dry running
$DIR/src/os/user/user_test.go: mutilating
$DIR/src/os/user/user_test.go: `userBuffer = old` passed
$DIR/src/os/user/user_test.go: `userBuffer = 1` passed
$DIR/src/os/user/user_test.go: `checkUser(t)` passed
$DIR/src/os/user/user_test.go: `checkUser(t)` passed
$DIR/src/os/user/user_test.go: `groupBuffer = old` passed
$DIR/src/os/user/user_test.go: `groupBuffer = 1` passed
$DIR/src/os/user/user_test.go: `checkGroup(t)` passed
$DIR/src/os/user/user_test.go: `checkGroupList(t)` passed
$DIR/src/os/user/user_test.go: `.GroupIds()` nonbuildable
